### DISCUSSION TITLE
account-banned?

### DISCUSSION THREAD

**Discussion by Guptaashish2003** [vercel/community#4314] [Account Issue]
### Question

Why my Account is banned when i'm login using my github account

**Comment by amyegan** [comment]
Sorry you're having trouble with your account. We can't safely discuss account information in public because of security concerns. You may have received an email with more details about the error and instructions on next steps. 

For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
